code/modules/medical/genetics/bioEffects/harmful.dm

```diff
@@ -791,12 +795,12 @@
 	var/run = 1
 	icon_state  = "bad"
 
-	OnLife()
+	OnLife(var/mult)
 		if (ishuman(owner))
 			var/mob/living/carbon/human/H = owner
 
 			if (H.blood_volume > 400 && H.blood_volume > 0)
-				H.blood_volume -= 2
+				H.blood_volume -= 2*mult
```

Comment:
This and polycythemia seem like an order of operations issue.  Unsure if this is a real issue though.

Reply:
this just extends the possible difference from 2 away from the thresholds to 2*mult away from the thresholds from what I can tell.

Reply:
this is using lifeprocess mult, not life mult, so it'll be capped at 20u anyways.